### DISCUSSION TITLE
The current OSX apple version of macOS Venture 13.0.1

### DISCUSSION THREAD

**Discussion by ClausKlein** [actions/virtual-environments#6640] [General questions]
Most packaged compiling with macOS-latest have a lot of deprecated warnings on my native OSX!

I use on my machine:
```
Apple clang version 14.0.0 (clang-1400.0.29.202)
Target: x86_64-apple-darwin22.1.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

Is there a plan to support this Version?

**Comment by mikhailkoliada** [comment]
Hi @ClausKlein!

I believe xcode toolchain we provide on macos-latest at the moment must support ventura as well (even though latest is Monterey right now). If you are interested in Ventura itself, please track https://github.com/actions/runner-images/issues/6426

**Reply by ClausKlein** [reply]
The `Apple clang version 14.0.0` generates a lot deprecated warnings that does not appear with **macOS-latest**!
